src/api/spec/support/capybara.rb

```diff
@@ -16,15 +16,20 @@
 
 # Automatically save the page a test fails
 RSpec.configure do |config|
+  config.before(:suite) do
+    html_files = Dir.glob(File.join(Capybara.save_path, '*.html'))
+    png_files = Dir.glob(File.join(Capybara.save_path, '*.png'))
+    FileUtils.rm(html_files + png_files)
```

Comment:
:thought_balloon: why not just remove/store all files?

Reply:
We can certainly do that, however, I did not want to change the behavior as I don't know what else gets stored in the capybara directory

Reply:
Just this...